e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -396,16 +396,15 @@ describe("scenarios > dashboard > tabs", () => {
     getDashboardCard(0).within(() => {
       cy.findByTestId("loading-spinner").should("exist");
       cy.wait("@saveCard");
-      cy.findByText("14").should("exist");
+      cy.findAllByTestId("table-row").should("exist");
     });
 
     // Loader in the 1st tab
     goToTab("Tab 1");
     getDashboardCard(0).within(() => {
       cy.findByTestId("loading-spinner").should("exist");
       cy.wait("@saveCard");
-      cy.findByText("58.4") // this is total from the first row
-        .should("exist");
+      cy.findAllByTestId("table-row").should("exist");
```

Comment:
Instead of checking for a particular value, which was no good anyways, we check for the existence of table rows.